blog/_posts/2022-01-17-scala-3.1.1-released.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.1.1 and forward compatibility news
+---
+
+Hello from the Scala 3 team! After a slightly longer than usual pause, we are back with a new release of the Scala compiler and exciting news about improvements in forward compatibility. The delay in the release was caused by urgent bugs that needed backporting and testing, and then the holiday season.
```

Comment:
I think you could omit the material about timing. It's normal for releases to slip a bit, especially around the holidays. People are primarily interested in the content of the release; few care whether it's a bit late.

If you feel you must account for the lateness, consider doing so much nearer end of the announcement, rather than the beginning. Leading with this sets a defensive, things-aren't-going-well tone. But things actually are going well (the delay is unimportant), so there's no need to set that tone.